docs/ipython_kernel_config.py

```diff
@@ -0,0 +1,2 @@
+c.InlineBackend.figure_formats = {'svg'}
+c.InlineBackend.rc = {'figure.dpi': 96}
```

Comment:
Why do we need this in both places? Here and in conf.py as `nbsphinx_execute_arguments`? Can we have them only in the conf.py?

Reply:
Sorry, this is mentioned in the docs you linked. Although, from this paragraph,

> To get SVG and PDF plots for nbsphinx, use something like this in your conf.py file:

I'd say that this is only required in `conf.py`, but I may be misunderstanding something. I'm not getting what is the `ipython_kernel_config.py` for in this case.

Reply:
The `ipython_kernel_config.py` is for JupyterLab, the stuff in `conf.py` is for `nbsphinx`.

It would be great if `nbsphinx` could also use `ipython_kernel_config.py`, but I don't know how to do that. TBH, I haven't looked into that yet. I *think* that `nbconvert` doesn't support that. 
But if I'm wrong, it should be simple to implement.

OTOH, you might want different settings for `figure_formats` ...
In this PR I selected only SVG for JupyterLab, but SVG + PDF for `nbsphinx`.
If you want to be able to export a PDF from JupyterLab, you should add PDF to the `figure_formats`.